scripts/render-rule-coverage-gif.py

```
@@ -134,7 +140,7 @@ def classify(ruleset, asserted, verified, probe, exclude, have_metrics, observed
     return order, state
 
 
-def draw(order, state, lit, title, agent, suite_labels, note, probes, exclude, size):
+def draw(order, state, lit, title, agent, suite_labels, note, probes, exclude, size, lit_seq):
```

Ruff (0.16.0)

[warning] Too many statements (57 > 50)

(PLR0915)